storio-content-resolver/src/main/java/com/pushtorefresh/storio/contentresolver/operations/delete/PreparedDeleteByQuery.java

```diff
@@ -47,7 +48,11 @@
     @NonNull
     @Override
     public DeleteResult executeAsBlocking() {
-        return deleteResolver.performDelete(storIOContentResolver, deleteQuery);
+        try {
+            return deleteResolver.performDelete(storIOContentResolver, deleteQuery);
+        } catch (Throwable throwable) {
```

Comment:
Hm, you think we should wrap all `Throwables` into `StorIOException`? I'd expect only `Exceptions`. Imagine if poorly written app used a lot of memory and performing operation in StorIO produces `OutOfMemoryError` — I don't want to see reports about such problems in our issues tracker :smile: 

Let's catch only `Exception`?